### ISSUE TITLE
TextField: Narrator does not read the error message on Edge's Latest Version

### ISSUE THREAD

**Issue by ssingh28** [OfficeDev/office-ui-fabric-react#5283]
MS Fabric React TextField is not working properly on SPO tenants default serach page .../_layouts/15/search.aspx on Edge with Narrator ON. It is not reading the error message related to a mandatory textfield when using keyboard someone tries to navigate to the textfield and then jumps to the next field. This problem seems to be only in the latest version of the Edge Microsoft Edge 42.17134.1.0. This does not arise in the older version of Edge 41.16299.492.0. This is happening on microsoft tenant.

**Comment by JasonGore** [comment]
This is also caused by lack of labeling. I've added a note to #5283 to make note of this issue, so I will close this as a duplicate.

**Comment by JasonGore** [comment]
Reopening as #5282 was closed as a Narrator issue.
